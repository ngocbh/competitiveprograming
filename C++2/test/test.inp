3
3
1 2 3
10
1 1 2 0 3 3 3 3 8 74
10
1 2 3 4 5 6 7 8 9 10
