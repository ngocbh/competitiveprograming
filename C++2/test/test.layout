<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<CodeBlocks_layout_file>
	<ActiveTarget name="Debug" />
	<File name="test.inp" open="0" top="0" tabpos="1" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="63" topLine="0" />
		</Cursor>
	</File>
	<File name="test.out" open="1" top="0" tabpos="6" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="11" topLine="0" />
		</Cursor>
	</File>
	<File name="test.cpp" open="1" top="1" tabpos="5" split="0" active="1" splitpos="0" zoom_1="0" zoom_2="0">
		<Cursor>
			<Cursor1 position="1096" topLine="22" />
		</Cursor>
	</File>
</CodeBlocks_layout_file>
