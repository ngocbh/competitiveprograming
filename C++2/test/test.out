1 3 2 
4 9 3 1 2 5 6 7 8 10 
1 9 5 3 7 2 10 6 4 8 
